certification/internal/policy/operator/scc_info.go

```diff
@@ -0,0 +1,124 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/bundle"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+)
+
+// securityContextConstraintsCheck evaluates the csv and logs a message if a non default security context constraint is
+// needed by the operator
+type securityContextConstraintsCheck struct {
+	customSCCSpecified  bool
+	builtInSCCSpecified bool
+}
+
+func getDefaultSCCs() []string {
```

Comment:
Nit: Can probably lose the 'get' here.

Reply:
Or remove this entirely, based on my further comment.

Reply:
removed 👍